superset/assets/.eslintrc

```diff
@@ -38,5 +38,8 @@
     "react/no-unescaped-entities": 0,
     "react/no-unused-prop-types": 0,
     "react/no-string-refs": 0,
+    "indent": 0,
```

Comment:
I assume these are due to the new `eslint` version which flagged them?

Reply:
yeah exactly